python/tenzir/src/tenzir/_bin/__init__.py

```diff
@@ -0,0 +1 @@
+# Internal helpers for locating and invoking packaged binaries.
```

Comment:
Lets keep these empty, afaik when there's any content the file has to be parsed and compiled (maybe even a .pyc is generated?)